src/Sulu/Bundle/MediaBundle/Resources/public/js/services/image-editor.js

```diff
@@ -12,9 +12,16 @@ define(['config', 'jquery'], function(Config, $) {
     'use strict';
 
     var featherEditor;
+    var featherUrl;
 
     if (!!Config.get('sulu-media').adobeCreativeKey) {
-        require(['//feather.aviary.com/imaging/v3/editor.js'], function() {
+        if (location.protocol === 'https:') {
+            featherUrl = 'https://dme0ih8comzn4.cloudfront.net/imaging/v3/editor.js';
```

Comment:
i was asking myself: where did he get this cryptic url? (: found after a short google search here https://creativesdk.zendesk.com/hc/en-us/articles/205045689-Image-Editor-Web-HTTPS-URL-for-the-editor-js